CHANGELOG.md

```diff
@@ -8,6 +8,55 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [1.2.0] - 2018-11-12
+
+### Added
+- New `WordPress.PHP.TypeCasts` sniff to the `WordPress-Core` ruleset.
+    This new sniff checks that PHP type casts are:
+    * lowercase;
+    * short form, i.e. `(bool)` not `(boolean)`;
+    * normalized, i.e. `(float)` not `(real)`;
+   Additionally, the new sniff discourages the use of the `(unset)` and `(binary)` type casts.
+- New `WordPress.Utils.I18nTextDomainFixer` sniff which can compehensively replace/add `text-domain`s in a plugin or theme.
+    Important notes:
+    - This sniff is disabled by default and intended as a utility tool.
+    - The sniff will fix the text domains in all I18n function calls as well as in a plugin/theme `Text Domain:` header.
+    - Passing the following properties will activate the sniff:
+        - `old_text_domain`: an array with one or more (old) text domain names which need to be replaced;
+        - `new_text_domain`: the correct (new) text domain as a string.
+- The `WordPress.NamingConventions.PrefixAllGlobals` sniff will now also verify that namespace names use a valid prefix.
+    * The sniff allows for underscores and (other) non-word characters in a passed prefix to be converted to namespace separators when used in a namespace name.
+        In other words, if a prefix of `my_plugin` is passed as a value to the `prefixes` property, a namespace name of both `My\Plugin` as well as `My_Plugin\\`, will be accepted automatically
+    * Passing a prefix property value containing namespace separators will now also be allowed and will no longer trigger a warning.
+- `WordPress` to the prefix blacklist for the `WordPress.NamingConventions.PrefixAllGlobals` sniff.
```

Comment:
Nit: Preclude the exception with `Add`, i.e.:
```suggestion
- Add `WordPress` to the prefix blacklist for the `WordPress.NamingConventions.PrefixAllGlobals` sniff.
```

Reply:
The  current way of writing this is in line with the way it has been in the previous changelogs, this bullet is in the `Added` section, so the `Add(ed)` at the start of the change item is left out.

Reply:
Cool 😎